### ISSUE TITLE
MapScale and Style in Version 4.7.0

### ISSUE THREAD

**Issue by MichaelEwe** [ClemensFischer/XAML-Map-Control#18]
I think, I have found a small issue with the WPF nuget package. Using a `MapScale` object  and setting its properties like this:
```
<map:MapScale Foreground="Blue" Stroke="Blue" StrokeThickness="2"/>
```
![image](https://user-images.githubusercontent.com/2787350/38058312-b0a416b8-32e2-11e8-9bc7-2f24f14ebf83.png)

works as expected. You get a blue label and a blue scale line with a thickness of 2.
If you do the same thing with a style like this:
```
<Style TargetType="map:MapScale" x:Key="S">
     <Setter Property="Stroke" Value="Blue"/>
     <Setter Property="StrokeThickness" Value="2"/>
     <Setter Property="Foreground" Value="Blue"/>
</Style>

<map:MapScale Style="{StaticResource S}"/>
```
![image](https://user-images.githubusercontent.com/2787350/38058398-fd184c8a-32e2-11e8-83b2-47376c525da7.png)

you get a blue label, and a line thickness of 2.
**But the scale line will not be blue. It will render with the `Foreground` brush of the enclosing `Map` object.**

I really have no clue on this one. Keep up this splendid work!

**Comment by ClemensFischer** [comment]
Interestingly it does work with a default Style, i.e. one without an `xKey`. I'll have a look.

**Comment by ClemensFischer** [comment]
Fixed with 4.7.1

**Comment by MichaelEwe** [comment]
You helped me a lot. Thank you very much.
